setup.py

```diff
@@ -61,7 +61,8 @@
         "networkx >= 2.5, ==2.*",
         "numpy ==1.*",
         "packaging >=19.2",
-        "pandas >=1.0.0, ==1.*",
+        # FIXME: consider pinning pandas==2.* if we can't support both v1 and v2 at the same time
+        "pandas >=1.0.0",
```

Comment:
There's also going to be pandas 3 soon: https://pandas.pydata.org/docs/dev/whatsnew/v3.0.0.html